src/Logger.ts

```diff
@@ -106,10 +106,10 @@ export default class Logger {
         this.oldLog(message, ...optionalParams);
 
         try {
-            let chunks = `[${(new Date()).toUTCString()}] Log: ${message.toString()}`.match(/.{1,2000}/g) || [message.toString()];
+            let chunks = `[${(new Date()).toUTCString()}] Log: ${message.toString()}`.match(/.{1,2000}/sg) || [message.toString()];
```

Comment:
What does this do?

Reply:
Adding the s flag in allows `.` to match newlines